examples/platform/esp32/time/TimeSync.cpp

```diff
@@ -94,9 +99,9 @@ void Init(const char * aSntpServerName, const uint16_t aSyncSntpIntervalDay)
     ChipLogProgress(DeviceLayer, "Initializing SNTP. Using the SNTP server: %s", sSntpServerName);
     esp_sntp_setoperatingmode(SNTP_OPMODE_POLL);
     esp_sntp_setservername(0, sSntpServerName);
-    esp_sntp_set_sync_interval(kSecondsInADay * aSyncSntpIntervalDay);
-    esp_sntp_init();
+    esp_sntp_set_sync_interval(kMilliSecondsInADay * aSyncSntpIntervalDay);
```

Comment:
This API takes milliseconds as the input, and not seconds?

Reply:
Yes